analyzers/src/SonarAnalyzer.Common/Helpers/CodeCopiedFromAttribute.cs

```diff
@@ -0,0 +1,42 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2024 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.Helpers;
```

Comment:
I suggest to find a better name and place for this attribute.

I don't have super ideas, but "Helpers" is a code smell and having a folder that contains both "meta-programming" and "functional helpers" is not great

Reply:
Following the offline discussion we had, the attribute has been removed from this PR, since we could not make the coverage work correctly with it, and we want to prioritize the delivery of this PR, as other PRs depend on this.
@andrei-epure-sonarsource I am resolving this discussion.